src/scripts/admin/filters/button-enhanced.js

```diff
@@ -84,10 +86,10 @@ const addInspectorControl = createHigherOrderComponent((BlockEdit) => {
 									label: __('Default', 'bcgov-block-theme'),
 									value: 'regular',
 								},
-								{
-									label: __('Small', 'bcgov-block-theme'),
-									value: 'small',
-								},
+								// {
```

Comment:
if we are not going to use the small variant we should probably just remove, doesn't seem like that big of thing to add back in.